components/layout_2020/style_ext.rs

```diff
@@ -313,10 +313,10 @@ impl ComputedValuesExt for ComputedValues {
         let border = self.get_border();
         flow_relative::Sides::from_physical(
             &PhysicalSides::new(
-                border.border_top_width.0,
-                border.border_right_width.0,
-                border.border_bottom_width.0,
-                border.border_left_width.0,
+                border.border_top_width.0.round_to_pixels(),
+                border.border_right_width.0.round_to_pixels(),
+                border.border_bottom_width.0.round_to_pixels(),
+                border.border_left_width.0.round_to_pixels(),
```

Comment:
This code has the same issue, I think. I do wonder why you need to round in both places though...shouldn't rounding in one be enough?